source/VideoPlayer.js

```diff
@@ -1873,7 +1877,7 @@
 		*/
 		timeUpdate: function(sender, e) {
 			//* Update _this.duration_ and _this.currentTime_
-			if (!e && e.srcElement) {
+			if (!e && e.srcElement || e.currentTime === undefined) {
```

Comment:
Do we care about short-circuiting this function if `e.currentTime` is `null` here as well - we could relax the conditional and make it `e.currentTime == null`.